src/object.d

```diff
@@ -180,7 +180,30 @@ class Object
     /**
      * Test whether $(D this) is equal to $(D o).
      * The default implementation only compares by identity (using the $(D is) operator).
-     * Generally, overrides for $(D opEquals) should attempt to compare objects by their contents.
+     * Generally, overrides and overloads for $(D opEquals) should attempt to compare objects by their contents.
+     * You will most likely want to add an overload that takes your specific type as the argument
```

Comment:
I don't mean to ponder upon petty details, but this is public documentation so it is best if we avoid having a casual tone.
"You will most likely want to add an overload ..." does not have the neutral, professional tone that is expected from technical prose. I would rephrase this paragraph as to get rid of the uses of "you". Something along the lines of: "The most common case is represented by overloads that take a specific type as argument ...".

Reply:
This is fine. The documentation isn't a textbook, and in fact some of the best textbooks are the ones that *do* take mostly conversational tone.

https://llvm.org/docs/CodingStandards.html contains almost 70 "you"s, for example and does it's job as documentation just fine despite actually being aimed at compiler engineers in their ivory towers.

What I will say, however, is that we should probably strive for listing a few (not just one) representative scenarios rather than going for "you will most likely". Rather than squabbling over the writing *style*, we need to ensure we are actually transmitting useful information. Well, squabble a bit, but the things that really matter are not the writing style when it comes to the documentation in this area of the *library* (the language specification is another matter).

Reply:
I slightly reworded it but that's not my writing style and I'm not going to pretend it is.

If you prefer, I can just delete the documentation.